app/konnect/dev-portal/customization/appearance.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Appearance
+no_version: true
+---
+
+The Dev Portal can be customized by those with Admin roles via the Appearance settings in the {{site.konnect_short_name}} Admin UI. To access the Appearance settings, click **Dev Portal** then **Appearance**.
+
+Here, you will have the ability to modify the following:
+
+* Logos: default logo and fav icon
+* Home Page Header: welcome message, primary header, and header image
+* Themes
+* Colors: background, text, and button colors
+* Fonts
+
```

Comment:
```suggestion

For details on the requirements for each customizable option, hover over the information (`i`) icon next to each item in the UI.

```
Suggest adding this line so that no one comes to us asking about guidelines, because it will happen.